css/media.css

```diff
@@ -0,0 +1,388 @@
+@media (max-width: 1220px) {
+   .welcome-page {
+     width: 100%;
+     margin: 0 auto;
+   }
+ }
+
+ @media (max-width: 1036px) {
+   .header__menu_link {
+     font-size: 12px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)